TgBot.java

```diff
@@ -0,0 +1,156 @@
+import org.telegram.telegrambots.bots.TelegramLongPollingBot;
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.methods.send.SendPhoto;
+import org.telegram.telegrambots.meta.api.objects.InputFile;
+import org.telegram.telegrambots.meta.api.objects.Update;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.InlineKeyboardMarkup;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.ReplyKeyboardMarkup;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.buttons.InlineKeyboardButton;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.buttons.KeyboardRow;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class TgBot extends TelegramLongPollingBot {
+
+
+    @Override
+    public void onUpdateReceived(Update update) {
+        String chatId = update.getMessage().getChatId().toString();
+        String message = update.getMessage().getText();
+
+
+        switch (message) {
+            case "Main Page", "/start", "Shop's Info" -> firstKeyboard(chatId, message);
+            case "Go Shopping", "Forward", "Back" -> keyboard(chatId, message);
+        }
+    }
+
+    public synchronized void sendPhoto(String chatId, long pageCounter) {
+
+        SendPhoto photo = new SendPhoto();
+        photo.setChatId(chatId);
+        if (pageCounter == 1) {
+            photo.setPhoto(new InputFile("https://imgur.com/a/dIc6rlc"));
+            photo.setCaption("Photo of product 1");
+        } else if (pageCounter == 2) {
+            photo.setPhoto(new InputFile("https://imgur.com/gallery/WaiBLIE"));

```

Comment:
I would recommend to store the image under the resource folder. There are at least 2 reasons why: 

- You become dependent on the availability of the image hosting server.
- It is much faster to load the image from a local drive.